blazor/dropdown-list/style.md

```diff
@@ -7,10 +7,66 @@ control: DropDown List
 documentation: ug
 ---
 
-# Style and Appearance in Blazor DropDown List Component
+# Style and Appearance in Dropdown List
 
 The following content provides the exact CSS structure that can be used to modify the control's appearance based on the user preference.
 
+## Read-only mode
+
+You can specify the boolean value to [Readonly](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Readonly) whether the DropDownList allows the user to change the value.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/style/readonly-mode.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Readonly mode](./images/style/blazor_dropdown_readonly-mode.png)
+
+## Disabled state
+
+You can specifies the boolean value to [Enabled](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_Enabled) property that indicates whether the component is enabled or not.
```

Comment:
Specify the boolean value to the [Enabled](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_Enabled) property that indicates whether the component is enabled or not.